### PR TITLE
Remove bedtime api

### PR DESCRIPTION
**Pull request by raymondjacobson**
Requires https://github.com/AudiusProject/audius-protocol/pull/6757 to be merged & deployed first

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -22,33 +20,6 @@ const app = express()
...
-    getBedtimeResponse(BedtimeFormat.TRACK, req, res)
-  }
-)
-
-router.get(
-  ['/embed/api/collections/:id', '/embed/api/collections/hashid/:hashId'],
-  (req: express.Request, res: express.Response) => {
-    getBedtimeResponse(BedtimeFormat.COLLECTION, req, res)
-  }
-)
```
Comment (Kyle-Shanks) [inline src/index.ts:39]: do we want to remove the track and collection endpoints also? or just collectibles?

Comment (raymondjacobson) [reply src/index.ts:39]: they're unused @nicoback2 @nicoback  moved these to `@audius/sdk` a bit ago

Comment (Kyle-Shanks) [reply src/index.ts:39]: ah nice

**Review by Kyle-Shanks — APPROVED** [review]
Rip Bedtime

**Pull request merged**